sdg-nuc-01

```diff
@@ -15,6 +15,9 @@ pluto:
   - netbox_api_token: 0123456789abcdef0123456789abcdef01234567
   network-config:
   - dutip: 192.168.2.1
+  jira-config:
```

Comment:
Consider adding these fields on the board-config section  instead. Probably these values will be utilized on other use cases aside from jira.

Reply:
Including jira-config to board-config makes sense. See da542a428b686f35cb30a45427e19063a036d1a7.